ct.sh

```diff
@@ -50,16 +50,45 @@ parse_command_line() {
 }
 
 install_chart_testing() {
-    echo "Installing chart-testing..."
+    if [[ ! -d "$RUNNER_TOOL_CACHE" ]]; then
+        echo "Cache directory '$RUNNER_TOOL_CACHE' does not exist" >&2
+        exit 1
+    fi
 
-    curl -sSLo ct.tar.gz "https://github.com/helm/chart-testing/releases/download/$version/chart-testing_${version#v}_linux_amd64.tar.gz"
-    tar -xzf ct.tar.gz
-    sudo mv ct /usr/local/bin/ct
-    mkdir "$HOME/.ct"
-    mv etc/* "$HOME/.ct"
+    local cache_dir="$RUNNER_TOOL_CACHE/ct/$version/amd64"
```

Comment:
nit: Should maybe lookup architecture, although I guess close to all run on linux 64

Reply:
done